owl_msgs/msg/ArmEndEffectorForceTorque.msg

```diff
@@ -0,0 +1,4 @@
+# Arm force-torque sensor force and torque
+
+Header header
+float64[6] value # force(N) [0-2], torque(N-m) [3-5], end effector frame
```

Comment:
Minor: I think this would be a little easier to read with the comment on the preceding line.